content/influxdb/v2/admin/security/enable-tls.md

```diff
@@ -176,3 +255,56 @@ update the following `influxdb_v2` output settings in your Telegraf configuratio
 ```
 
 Restart Telegraf using the updated configuration file.
+
+## Troubleshoot TLS
+
+### 1. Check InfluxDB logs
```

Comment:
Is there a specific order to these troubleshooting steps? I've generally moved away from including these types of ordered numbers in headings, but I won't block the PR for these.

Reply:
> I've generally moved away from including these types of ordered numbers in headings,

I've noticed and so I try to avoid them as well. I don't think there's a necessary order in this case; I'll remove them.